tests/zfs-tests/tests/functional/cli_root/zfs_destroy/zfs_destroy_003_pos.ksh

```diff
@@ -129,25 +129,25 @@ for clone in $child_fs1_snap_clone $child_fs1_snap_clone1; do
 done
 
 log_note "Verify that 'zfs destroy -r' fails to destroy dataset " \
-	"with clone dependant outside it."
+	"with clone dependent outside it."
```

Comment:
Just noting that the changes dependant -> dependent in this file seems to be BrE -> AmE changes.
Would it be better with 'dependent clone' than 'clone dependent'?